cypress/support/step_definitions/testSetup.js

```diff
@@ -106,7 +106,14 @@ function destroyAppInstance(testType) {
     try {
       cy.sendMessagetoApp(requestTopic, responseTopic, intentMessage).then((response) => {
         if (response != CONSTANTS.NO_RESPONSE) {
-          fireLog.assert(false, 'App failed to unload, Reason: ' + closeReason);
+          fireLog.log(false, 'App failed to unload, Reason: ' + closeReason);
+              const requestMap = {
+                method: CONSTANTS.REQUEST_OVERRIDE_CALLS.UNLOADAPP
+              };
+              cy.sendMessagetoPlatforms(requestMap).then(() => {
+              //Config modules needs override for validation of app unload
+              cy.log("Config module execution complete to unload app"); 
```

Comment:
If I were to read this log while running tests or debugging I wouldn't really understand its significance. Could you rework it so that it is more meaningful?

Reply:
@jbigel Does one of these log message seems more correct? if not can you suggest? 'Platform's unload app execution complete' / 'Config module unload app execution complete'

Reply:
The first sounds good to me, thanks.

Reply:
fixed the log message, Can you please take a look?